assets/css/map/controls/_layers_control.scss

```diff
@@ -0,0 +1,39 @@
+.c-layers-control {
+  display: flex;
+  border: none !important;
+  gap: 5px;
+}
+
+.c-layers-control__button {
+  @include button-icon(20px);
+  background-color: $white;
+  flex: 0;
+  padding: 5px;
+  span svg {
+    stroke: none;
+  }
+
+  &--selected {
+    background-color: $color-gray-200;
+  }
+  &:focus-visible {
+    outline: revert;
```

Comment:
question(non-blocking): what are we `revert`ing here to, the user-agent style?

Reply:
Yep! The other controls also have the user-agent focus style, so reverting to undo the https://github.com/mbta/skate/blob/400edc0adcb79897b99a2f24c3fbd48b99a22881/assets/css/_base.scss#L14 for buttons.